engine/src/main/java/org/terasology/config/flexible/FlexibleConfigManagerImpl.java

```diff
@@ -87,21 +87,10 @@ public void saveAllConfigs() {
         }
     }
 
-    private void ensureDirectoryExists(Path filePath) {
-        try {
-            Files.createDirectories(filePath.getParent());
-        } catch (Exception e) {
-            throw new RuntimeException("Cannot create directory for FC!");
-        }
-    }
-
     private Path getPathForFlexibleConfig(SimpleUri flexibleConfigId) {
-        Path filePath = PathManager.getInstance()
-                                        .getHomePath()
-                                        .resolve("configs")
-                                        .resolve(flexibleConfigId.getModuleName().toString())
-                                        .resolve(flexibleConfigId.getObjectName().toString() + ".cfg");
-        ensureDirectoryExists(filePath);
```

Comment:
What's the reasoning behind the removal of this check? Is the directory guaranteed to exist from some other parts of the code?

Reply:
Yes.

https://github.com/MovingBlocks/Terasology/pull/3252/files#diff-5bbcd1d375372f183a3afd8dafa16318R274